src/gui/tracks/TrackView.cpp

```diff
@@ -130,23 +130,40 @@ TrackView::TrackView( Track * track, TrackContainerView * tcv ) :
  */
 void TrackView::resizeEvent( QResizeEvent * re )
 {
-	if( ConfigManager::inst()->value( "ui",
-					  "compacttrackbuttons" ).toInt() )
+	m_trackOperationsWidget.setFixedSize(getTrackFixedOperationsWidth(), height() - 1);
+	m_trackSettingsWidget.setFixedSize(getTrackFixedSettingsWidth(), height() - 1);
+	m_trackContentWidget.setFixedHeight(height());
+}
+
+
+const int TrackView::getTrackFixedWidth()
+{
+	return getTrackFixedSettingsWidth() + getTrackFixedOperationsWidth();
+}
+
+const int TrackView::getTrackFixedSettingsWidth()
```

Comment:
This function can be rewritten like this. The comments are also very unnecessary IMO: The code is clear enough that one can already tell what is happening.

```cpp
const int TrackView::getTrackFixedSettingsWidth()
{
	const auto isCompact = ConfigManager::inst()->value("ui", "compacttrackbuttons").toInt();
	return isCompact ? DEFAULT_SETTINGS_WIDTH_COMPACT : DEFAULT_SETTINGS_WIDGET_WIDTH;
}
```